src/server/client-data-mapper.js

```diff
@@ -55,6 +55,38 @@ class ClientDataMapper {
     });
   }
 
+  async mapImportBatches(batches, user) {
+    const idSet = new Set(batches.map(batch => batch.createdBy));
```

Comment:
I think `userIdSet` would be better, I thought initially it's the batches Ids.

Reply:
This was fixed, I forgot to comment on it